config/rollup.iife.config.js

```diff
@@ -19,14 +19,17 @@ export const O_RDONLY = ${O_RDONLY}
 export default {
     input: "main.js",
     output: {
-        file: "build/snarkjs.js",
+        file: "build/snarkjs.iife.js",
```

Comment:
```suggestion
        file: "build/snarkjs.js",
```
let's keep this since it is breaking if people are reaching into this path